portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,110 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>Kayla Leung Portfolio</title>
+    <title>Kayla Leung</title>
+
     <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="font.css">
+
     <script src="script.js"></script>
+    <script src="https://kit.fontawesome.com/b99e675b6e.js"></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    
+
   </head>
   <body>
-    <div id="content">
-      <h1>Kayla Leung</h1>
-      <p>This is my portfolio.</p>
-      <img src="images/kaylaleung.jpeg"/>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+
+    <!-- Toggle Side View -->
+    <input type="checkbox" name="">
+    <span class="icon"></span>
+    
+    <!-- Left Side Bar -->
+    <div class="sidebar">
+
+        <img src="images/kaylaleung.jpeg" style="width:175px;height:175px;" alt="user profile photo">
+
+        <div>
+          <h2>Kayla Leung</h2>
+          <h4>Software Engineering Intern</h4>
+        </div>
+
+        <div class="social_media">
+          <ul>
+            <li><a href="//www.facebook.com/kaylayleung"><i class="fab fa-facebook-f" alt="facebook icon"></i></a></li>
+            <li><a href="//www.instagram.com/kaylaleung/"><i class="fab fa-instagram" alt="instagram icon"></i></a></li>
+            <li><a href="//www.linkedin.com/in/kaylaleung/"><i class="fab fa-linkedin" alt="linkedin icon"></i></a></li>
+            <li><a href="//www.github.com/kaylaleung"><i class="fab fa-github" alt="github icon"></i></a></li>
+          </ul>
+        </div>
+
+        <div class="menu">
+          <ul>
+              <li><a href="#home" class="active"><i class="fas fa-home"></i>Home</a></li>
+              <li><a href="#education"><i class="fas fa-user"></i>Education</a></li>
+              <li><a href="#experience"><i class="fas fa-address-card"></i>Experience</a></li>
+              <li><a href="#projects"><i class="fas fa-project-diagram"></i>Projects</a></li>
+              <li><a href="#blog"><i class="fas fa-blog"></i>Blog</a></li>
+          </ul>
+        </div>
     </div>
+
+    <!-- Main Content Section -->
+    <div id="home" class="menu">
+      <section>
+        <div class="header"></div>
```

Comment:
So this header contains nothing. I would try to avoid empty divs like this, especially when there are more semantic elements like <header> for this.

Generally the structure of a section> is (I had to remove some angle brackets since GitHub apparently escapes HTML poorly)

section>
  header>
    h2>About Me/h2>
  /header>
  Lorem Ipsum content
/section>